Seems that even if you do `./configure --without-kerberos`, configure 
will still search for some krb headers/libs, evil!

http://bugs.gentoo.org/show_bug.cgi?id=80811

--- configure.ac
+++ configure.ac
@@ -2464,7 +2464,6 @@
 		if test ! -z "$blibpath" ; then
 			blibpath="$blibpath:${KRB5ROOT}/lib"
 		fi
-	fi
 
 	AC_CHECK_HEADERS(gssapi.h gssapi/gssapi.h)
 	AC_CHECK_HEADERS(gssapi_krb5.h gssapi/gssapi_krb5.h)
@@ -2473,6 +2472,7 @@
 	LIBS="$LIBS $K5LIBS"
 	AC_SEARCH_LIBS(k_hasafs, kafs, AC_DEFINE(USE_AFS))
 	AC_SEARCH_LIBS(krb5_init_ets, $K5LIBS, AC_DEFINE(KRB5_INIT_ETS))
+	fi
 	]
 )
 
